consoleApp/src/main/java/by/issoft/consoleApp/handlers/top/TopCommand.java

```diff
@@ -0,0 +1,13 @@
+package by.issoft.consoleApp.handlers.top;
+
+import by.issoft.consoleApp.handlers.AppCommand;
+import by.issoft.store.Store;
+
+public class TopCommand implements AppCommand {
+    @Override
+    public void execute(Store store) {
+        prepareSortProductList(store, new ProductPriceDescComparator())
```

Comment:
use generic method that creates comportors

Reply:
Done